package.json

```
@@ -97,5 +97,6 @@
         "singleton": true
       }
     }
-  }
+  },
+  "packageManager": "yarn@4.7.0"
```

As commented above, we rely on `jlpm`

```suggestion
  }
```